content/tutorial-rippled-setup.md

```diff
@@ -57,16 +58,17 @@ There are several properties that define a good validator. The more of these pro
 
 At present, Ripple (the company) cannot recommend any validators aside from those in the default validator list. However, we are collecting data on other validators and building tools to report on their performance. For metrics on validators, see [validators.ripple.com](https://validators.ripple.com).
 
+{% include 'snippets/capacity-planning.md' %}
 
 # Installing rippled
 
 For development, you can [compile `rippled` from source](https://wiki.ripple.com/Rippled_build_instructions).
 
 Production `rippled` instances can [use Ripple's binary executable](#installation-on-centosred-hat-with-yum), available from the Ripple [yum](https://en.wikipedia.org/wiki/Yellowdog_Updater,_Modified) repository.
 
-## System Requirements
+## Minimum System Requirements
 
-A `rippled` server should run comfortably on commodity hardware, to make it inexpensive to participate in the network. At present, we recommend the following:
+A `rippled` server should run comfortably on commodity hardware, to make it inexpensive to participate in the network. At present, we recommend the following mimimum requirements:
 
 - Operating System:
     - Production: CentOS or RedHat Enterprise Linux (latest release) or Ubuntu (15.04+) supported
```

Comment:
Sort of unrelated, but let's update the minimum supported version of Ubuntu to 16.04 since 15.04 is now long past end-of-life.